app/javascript/leaflet/maps/search_results_map.js

```diff
@@ -0,0 +1,150 @@
+import { latLngBounds } from "leaflet";
+import GeoBlacklightViewerMap from "../viewers/map";
+import GeoSearchControl from "../controls/geosearch";
+import { bboxToBounds, geoJSONToBounds } from "../utils";
+
+export default class SearchResultsMap {
+  constructor(element) {
+    const data = element.dataset,
+      opts = { baseUrl: data.catalogPath },
+      world = latLngBounds([
+        [-90, -180],
+        [90, 180],
+      ]);
+    let bbox;
+
+    // StaticButton
+    // Create the anchor element
+    const staticButtonNode = document.createElement("a");
+    staticButtonNode.setAttribute("id", "gbl-static-button");
+    staticButtonNode.setAttribute("href", "#");
+    staticButtonNode.setAttribute("style", "display:none;");
+    staticButtonNode.className = "btn btn-primary";
+    staticButtonNode.textContent = "Redo search here";
+
+    // Create the span element for the glyphicon
+    const span = document.createElement("span");
+    span.className = "glyphicon glyphicon-repeat";
+
+    // Append the span to the anchor element
+    staticButtonNode.appendChild(span);
+
+    // DynamicButton
+    const dynamicButtonNode = document.createElement("label");
+    dynamicButtonNode.setAttribute("id", "gbl-dynamic-button");
```

Comment:
I am not sure we should be redoing the dynamic / static button everywhere unless the button is different from the default.